### ISSUE TITLE
Alert banners have role="alert" set

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#374]
In a previous discussion at https://github.com/localgovdrupal/localgov_alert_banner/issues/146, it was decided to remove `role="alert"` from alert banners. However, these are still being introduced:

![image](https://user-images.githubusercontent.com/19519457/219037435-a575f088-117a-43b1-9847-95871d6657c5.png)

I think this is through the base theme: https://github.com/localgovdrupal/localgov_base/blob/fd75598dd72af24d5fbe3c6fd956b4bd854565ca/templates/block/localgov-alert-banner.html.twig#L40

This was picked up in some initial accessibility testing of our site. Full details below:

> ### Impact
> All banner component variants have been marked-up as ‘alerts’. This means content within the banner – irrespective if its urgency – may be automatically announced for users. In addition, users navigating through the page will hear ‘alert’ for content which is purely informational.  
> 
> ### Best practice 
> The item does not present a major barrier but updating the component is recommended to reduce audible clutter for users and better differentiate between true alerts – which require immediate user attention / interaction – vs information which is important or useful to know but doesn’t necessarily require immediate action. 
> 
> Banners use role=”alert” in all instances. This role turns the content in to a live region. Live regions of this kind are intended as containers for content which changes dynamically – generally as a result of a user interaction – such as generating form errors. This live region is then used to automatically announce errors generated from a failed form submission. 
> 
> ### Suggested fix
> Recommend removing role=”alert” from all notification banners except those which require a user to be notified immediately (time sensitive) and/or which require a user response. An example being content notifying a user their session is about to time out. 
> 
> Note that with this banner uses `aria-role="region"` with an `aria-labelledby attribute which means the content will appear in a screen reader outline of page regions with a clear description. The inclusion of ‘important’ in the text content also helps make it clear the information should be noted.

**Comment by keelanfh** [comment]
@msayoung
